tests/staticfiles_tests/test_management.py

```diff
@@ -340,3 +340,13 @@ def test_broken_symlink(self):
         os.unlink(path)
         self.run_collectstatic()
         self.assertTrue(os.path.islink(path))
+
+    def test_clear_broken_symlink(self):
+        """
+        Test broken symlink gets deleted with ``--clear`` option.
```

Comment:
Just state expected behavior instead of "Test...", e.g. "collectstatic --clear should delete broken symlinks."